mmengine/visualization/visualizer.py

```diff
@@ -615,6 +625,7 @@ def draw_circles(
         line_widths: Union[Union[int, float], List[Union[int, float]]] = 2,
         face_colors: Union[str, tuple, List[str], List[tuple]] = 'none',
         alpha: Union[float, int] = 0.8,
+        zorder: float = 1.0,
```

Comment:
What is the reason for using `zorder=1.0` for `draw_circles` but `zorder=2.0` for `draw_boxes`